tools/third_party.py

```diff
@@ -49,7 +49,6 @@ def python_env(env=None, merge_env=None):
 # Run Yarn to install JavaScript dependencies.
 def run_yarn():
     node_modules_path = os.path.join(third_party_path, "node_modules")
-    # Note to keep the root directory clean, we keep package.json is in tools/.
```

Comment:
I clearly did not read the code in here well enough. The package.json is stored here for the third_party repo (why not store that in the third_party repo though?)